src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Haha, you kidding. Este is piece of art, just few extra CSS best practises would help resolve CSS mess in growing projects )